css-layout-classes/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>CSS Layout Classes</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-one-third">
+          <img src="./images/link.png" />
+        </div>
+        <div class="column-two-thirds">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>
+            Super Smash Bros. Ultimate received ultimate acclaim from both
+            critics and players, with some critics calling it the best
+            installment in the series. It received praise for its large amount
+            of content and fine-tuning of existing Smash gameplay elements,
+            although its online mode was widely critisized.
+          </p>
+        </div>
+        <div class="column=full">
+          <img src="./images/yoshi.png" />
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-one-quarter">
+          <img src="./images/kirby.png" />
+        </div>
+
+        <div class="column-two-quarter">
+          <img src="./images/fox.png" />
+        </div>
+
+        <div class="column-three-quarter">
+          <img src="./images/pikachu.png" />
+        </div>
+
+        <div class="column-four-quarter">
+          <img src="./images/ness.png" />
+        </div>
```

Comment:
All of these columns should have the same class name. Something like `column-fourth`